src/ast/codegen_llvm.cpp

```diff
@@ -661,87 +663,92 @@ void CodegenLLVM::visit(Call &call)
   }
   else if (call.func == "print")
   {
-    /*
-     * perf event output has: uint64_t asyncaction_id, uint64_t top, uint64_t div, string map_ident
-     * The asyncaction_id informs user-space that this is not a printf(), but is a
-     * special asynchronous action. The ID maps to print(). The top argument is either
-     * a value for truncation, or 0 for everything. The div argument divides the output values
-     * by this (eg: for use in nanosecond -> millisecond conversions).
-     * TODO: consider stashing top & div in a printf_args_ like struct, so we don't need to pass
-     * them here via the perfdata output (which is a little more wasteful than need be: I'm using
-     * uint64_t's to avoid "misaligned stack access off" errors when juggling uint32_t's).
-     */
+    auto elements = AsyncEvent::Print().asLLVMType(&b_);
+    StructType *print_struct = b_.GetStructType("print_t", elements, true);
```

Comment:
Maybe do same as below?
```suggestion
    StructType *print_struct = b_.GetStructType(call.func + "_t", elements, true);
```